scripts/include/common.sh

```diff
@@ -6,23 +6,28 @@ if [ -n "$VALUES_OVERRIDE" ] && [ -f "$VALUES_OVERRIDE" ]; then
   OVERRIDE=$(cat $VALUES_OVERRIDE)
 fi
 
-export cluster_name=${CLUSTER_NAME:-kind}
-export BUILD_DIR=build${cluster_name}
+export CLUSTER_NAME=${CLUSTER_NAME:-kind}
+export BUILD_DIR=build${CLUSTER_NAME}
+
+export KUBECONFIG=$PWD/$BUILD_DIR/kubeconfig
```

Comment:
Used absolute directory to let it work no matter what the current directory is in each script